src/context.rs

```diff
@@ -67,6 +67,16 @@ pub struct Context {
     pub quit_requested: bool,
 }
 
+impl Context {
+    /// Attempts to terminate the [`ggez::event::run()`](fn.run.html) loop by requesting a
+    /// [`quit_event`](EventHandler::quit_event) at the very start of the next frame. If this event
+    /// returns `Ok(false)`, then [`Context.continuing`](struct.Context.html#structfield.continuing)
+    /// is set to `false` and the loop breaks.
```

Comment:
Some of these links might have been broken by being moved. I'll try to update them with a quick patch this evening.